### DISCUSSION TITLE
Chain-Mode issue

### DISCUSSION THREAD

**Discussion by Xerror04** [bangcorrupt/hacktribe#326] [Q&A]
Hello, first of all, thank you for the "Hacktribe-OS", it's a nice firmware! However, I have the following issue. When I use "Chain to", there is a brief interruption when transitioning to the new pattern. It doesn't smoothly transition to the next pattern. I have the blue synthesizer.

**Comment by bangcorrupt** [comment]
The sounds will not carry over, so there can be a discontinuity but there shouldn't be a pause.

Is it different to when you change patterns manually?

Is it different to when using the factory firmware?

Can you share some pattern files that demonstrate the issue?

**Comment by Koalemosthegod** [comment]
I've only ever been able to get Reverb and Delay to carry over to the next pattern. You have to make sure the MFX settings are same on both patterns then, you'll get MFX carry over. With IFX, you have to have the same sound settings and have the sound on the same channel to get carry over.
